roles/openstack/neutron/defaults/main.yml

```diff
@@ -0,0 +1,17 @@
+# Network defaults
+
+external_network: False
+timeout_network: 180
+shared_network: False
```

Comment:
Please remove (`external`|`shared`)`_network` default. there's no reasonable use case were user might wish to create all networks as external/shared.
If we have such an uncommon user, he will simply need to set his template explicitly.